Celeste.Mod.mm/Patches/BackdropRenderer.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Microsoft.Xna.Framework.Graphics;
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+using Monocle;
+using MonoMod;
+using MonoMod.Cil;
+using MonoMod.Utils;
+using System;
+
+namespace Celeste {
+    class patch_BackdropRenderer : BackdropRenderer {
+        private bool usingSpritebatch;
+        
+        public void StartSpritebatchLooping(BlendState blendState) {
```

Comment:
this might cause problems if an `Effect` that wants a spritebatch renders after a `Backdrop`, and doesn't expect the spritebatch to be looping; this could be fixed by adding a `private bool spritebatchLooping` and checking that too, but that requires more patching

Reply:
Maybe I should just MonoModReplace the method at this point

Reply:
yea might as well
would be a good place to fix the bug with additive spritebatches persisting into the next backdrop too

Reply:
Well it sounded like maps have been made that use that behavior, so I'm not sure how to handle that one.

Reply:
Also MonoModReplacing definitely increases the probability of max having to rewrite their hooks.

Reply:
BackdropRenderer.Render logic has now been refactored to account for this.